packages/synapse-react-client/package.json

```diff
@@ -249,14 +243,11 @@
     "test": "jest",
     "test:ci": "jest --watchAll=false --coverage",
     "test:coverage": "jest --coverage",
-    "build": "rimraf dist && pnpm build:copy-assets && pnpm build:js && pnpm build:esbuild",
-    "build:js": "tsup",
-    "build:fix-extension": "mv dist/synapse-react-client.js dist/synapse-react-client.mjs",
+    "build": "rimraf dist && pnpm --parallel run \"/^build:.*/\"",
     "build:copy-assets": "copyfiles -u 1 \"src/**/*.{css,scss,svg}\" dist",
-    "build:esbuild": "node esbuild.config.mjs",
-    "build:docs": "pnpm storybook build",
-    "build:gh-pages": "pnpm storybook build && touch storybook-static/.nojekyll",
-    "build-storybook": "pnpm storybook build",
+    "build:js": "vite build",
+    "build:umd": "vite build -c vite.config.umd.ts",
+    "storybook:build": "pnpm storybook build && touch storybook-static/.nojekyll",
```

Comment:
Renamed `build:storybook` to `storybook:build` because it does not need to run at the same time as the other build scripts, which are now run in parallel and identified using the`build:` prefix